src/main/java/com/easyrest/components/BaseComponent.java

```diff
@@ -14,5 +16,8 @@ protected void enterData(String data, By element, WebDriver driver){
         driver.findElement(element).clear();
         driver.findElement(element).sendKeys(data);
     }
-
+    protected void clearWithKeysAndFillInput(By element, String inputText){
```

Comment:
explain please how this method work?

Reply:
This method highlights the line, clears it and inserts the text